File: Print sum 0-255/print_sum255.py
# A function, with a parameter int, that will print the consecutive sum of 0 to int and return the sum
def consecutive_sum(int):
    total = 0
    for i in range(int+1):
        total += i
        print(f"+ {i} = {total}")
    print(f"consecutive_sum returned {total}")
    return total

consecutive_sum(255)

File: Print sum 0-255/print_sum255.js
// A function, with a parameter int, that will print the consecutive sum of 0 to int and return the sum
function consecutive_sum(int){
    let sum = 0;
    for(i=0;i<=int;i++){
        sum+=i;
        console.log("+", i, "=", sum);
    }
    console.log("consecutive_sum returned", sum);
    return sum;
}

consecutive_sum(255);